eng/common/pipelines/templates/stages/archetype-sdk-tool-pwsh.yml

```diff
@@ -15,20 +15,20 @@ variables:
   - template: /eng/pipelines/templates/variables/globals.yml
 
 stages:
-  - stage: 'eng_script_tests'
+  - stage: Run_PowerShell_Tests
```

Comment:
You can set `displayName: Run PowerShell Tests` and leave the id name empty. That allows for better names without the need for reserved chars like _